app/models/manageiq/providers/google/cloud_manager/cloud_volume.rb

```diff
@@ -1,2 +1,7 @@
 class ManageIQ::Providers::Google::CloudManager::CloudVolume < ::CloudVolume
+  def self.params_for_create(_)
+    {
+      :fields => []
```

Comment:
@skateman if we don't `supports :cloud_volume_create` do we need to declare empty params_for_create?  Who is going to be asking for params without checking `supports?(:cloud_volume_create)` first?

Reply:
Well, we do support editing of the google cloud volumes and it tries to load the additional schema and without it defined it crashes. It seemed as a cleaner solution than to send two api requests, one asking for supporting create and one for the actual schema if it is supported...